roles/scalelite/templates/nginx/scalelite/greenlight.nginx.j2

```diff
@@ -0,0 +1,94 @@
+location /greenlight/cable {
+    proxy_pass http://{{ scalelite_greenlight_container | default('greenlight') }}:3000;
+    proxy_redirect off;
+    proxy_http_version 1.1;
+
+    proxy_set_header Host $host;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header X-Forwarded-Proto $scheme;
+
+    proxy_set_header Connection "upgrade";
+    proxy_set_header Upgrade $http_upgrade;
+}
+
+location /greenlight {
+    proxy_pass http://{{ scalelite_greenlight_container | default('greenlight') }}:3000;
+    proxy_redirect off;
+    proxy_http_version 1.1;
+
+    proxy_set_header Host $host;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header X-Forwarded-Proto $scheme;
+    proxy_set_header Connection "";
+    
+    # Set Headers
+    add_header Content-Security-Policy "default-src 'self'; script-src 'self'; connect-src 'self'; img-src 'self'; style-src 'self'; frame-ancestors 'self'; form-action 'self'; worker-src 'self'";
+    add_header Strict-Transport-Security "max-age=31556952; includeSubDomains";
+    add_header X-Content-Type-Options "nosniff";
+    add_header Cache-Control "max-age=0, private, must-revalidate";
+    add_header X-Frame-Options SAMEORIGIN;
```

Comment:
the line "add_header Set-Cookie "Path=/; Secure; HttpOnly; SameSite=Strict";" should be added, also done here: https://github.com/hpi-schul-cloud/infra-schulcloud/pull/1475/files. Tested if login to Greenlight still works and could not see any negative consequences. And could not find a variable in Greenlight with which to change cookie settings.

Reply:
Done